lib/storage/metric_name.go

```diff
@@ -481,6 +481,9 @@ func MarshalMetricNameRaw(dst []byte, labels []prompb.Label) []byte {
 		}
 		if len(label.Value) > maxLabelValueLen {
 			atomic.AddUint64(&TooLongLabelValues, 1)
+			logger.Warnf("truncated label value as it exceeds configured maximal label value length: max %d, actual %d;"+
```

Comment:
I think it should be limited logger. E.g. log only one message in 5 seconds.

But we have to be careful with `labelsToString` - it could be called for each logged line and decrease overall ingestion performance. 

 See realization for `trackDroppedLabels`.

Reply:
Thanks for suggestion, implemented this similar to `trackDroppedLabels`